### ISSUE TITLE
Can't instantiate Amanda::DB::Catalog2::SQLite: Cannot prepare: database is locked

### ISSUE THREAD

**Issue by bjne** [zmanda/amanda#161]
FAILURE DUMP SUMMARY:
  chunker: FATAL Cannot execute: database is locked at /usr/lib/amanda/perl/Amanda/DB/Catalog2.pm line
557.
  chunker: FATAL Can't instantiate Amanda::DB::Catalog2::SQLite: Cannot prepare: database is locked at
/usr/lib/amanda/perl/Amanda/DB/Catalog2/SQLite.pm line 111.

**Comment by djmitche** [comment]
This error typically occurs when the DB is already in use by another process.  Is that the case here?  Perhaps another Amanda run using the same DB?

**Comment by bjne** [comment]
No, this is not the case here. Only a single amdump process running

**Comment by djmitche** [comment]
OK, well, that was my only guess, sorry :)

**Comment by chassell** [comment]
I am guessing we have a problem with a stale lock that needs to be checked.
